lib/S3.js

```diff
@@ -30,22 +30,62 @@ var S3StorageProvider = function(options) {
     bucket: this._options.bucket,
     region: this._options.region
   })
-}
+
+this._cb = function(error, res, callback) {
```

Comment:
Could this field please have a more descriptive name?  `this._cb` doesn't tell me much about what it's supposed to do.